test/functional/testCases.js

```diff
@@ -542,6 +542,20 @@ const testCases = [
                                 object_id: '.1.0.0.1',
                                 name: 'psBatteryVoltage',
                                 type: 'Number'
+                            },
+                            {
+                                object_id: 't',
+                                name: 'value',
+                                type: 'Number'
+                            },
+                            {
+                                name: 'fireDetectionThreshold',
+                                type: 'Number'
+                            },
```

Comment:
What about?

```suggestion
                            {
                                name: 'fireDetectionThreshold',
                                type: 'Number'
                            },
                            {
                                object_id: 'fdt',
                                name: 'fireDetectionThreshold',
                                type: 'Number'
                            },
```

next

* Send mesuare fdt: 20 -> CB expectations fireDetectionThreshold: 20
* Send mesuare fireDetectionThreshold: 20 -> CB expectations fireDetectionThreshold: 20

Reply:
@fgalan exactly. By using the objectid and the name we should have the same result.

Reply:
> What donyou mean by it does not work?

@tzzed When I applied the suggested code done by fermin the test didn't pass.